### DISCUSSION TITLE
UX Changes

### DISCUSSION THREAD

**Discussion by tonygem2022** [iTwin/itwinjs-core#5759] [Q&A]
We are considering changing some of the visual appearance of the platform for our needs, including a consolidation of function buttons, moving buttons into more logical groupings etc. 

Notwithstanding further explanation, can someone please point us in the right direction as to how we may be able to edit the visual interface without having to go through a significant rewrite?

I recall seeing guidance on this type of effort being easier from a particular version but have struggled to find this to date.

What can we / cant we do readily?

Can we readily edit menus or existing tool bars at the top and bottom of screen for example ? e.g. can the messages in the bottom left be removed etc

Understanding what can be edited will help us develop a plan for preferred UX. Is this documented anywhere?

Thanks.

Tony

**Comment by NancyMcCallB** [comment]
Hi, @tonygem2022 ,

I'm so sorry we missed this discussion when you posted it and I apologize for the delay in response.

We have some [documentation ](https://www.itwinjs.org/learning/ui/augmentingui/)about how to change the UI in an AppUI viewer app, but it would help if you provided some more details about what you want to achieve. Do you have (or could you make) a mockup of the resulting UX/UI you want in your app? It can be as rough as a screen shot marked up in a simple image editor, but if we have you goal in mind, we can perhaps help you direct your efforts.
